app/controllers/users/two_factor_authentication_controller.rb

```diff
@@ -257,6 +257,13 @@ def track_events(otp_delivery_preference:, otp_delivery_selection_result:)
           phone_number: parsed_phone.e164,
           otp_delivery_method: otp_delivery_preference,
         )
+      else
+        irs_attempts_api_tracker.mfa_login_phone_otp_sent(
+          success: @telephony_result.success?,
+          phone_number: parsed_phone.e164,
+          otp_delivery_method: otp_delivery_preference,
+          failure_reason: irs_attempts_api_tracker.parse_failure_reason(@telephony_result),
+        )
```

Comment:
I am open to conversation on this - or removing it altogether.  It struck me that we might get through this logic without logging an event, when an event should always be logged here.

Reply:
I'm not sure I follow, do you mean adding this event is a duplicate log?

Reply:
No, just that getting to this `if`-`elsif` statement means that we should log some kind of an `mfa_phone_otp_sent` event.  On the face of it - it looks like we could conceivably _not_ log an event (because there is no `else`) - but in looking closer at the `UserSessionContext`, it seems that there are only three options and this `if`-`elsif` statement covers all three of them.  I'll remove this code addition. 👍